test/verify/check-journal

```diff
@@ -468,9 +476,13 @@ ExecStart=/bin/sh -c 'for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 1
 
     @skipImage("ABRT not available", "debian-stable", "debian-testing", "ubuntu-stable",
                "ubuntu-2004", "fedora-coreos", "rhel-8-3", "rhel-8-3-distropkg", "centos-8-stream")
+    @nondestructive
     def testAbrtSegv(self):
         b = self.browser
+        m = self.machine
 
+        m.execute("systemctl restart abrt-journal-core")
```

Comment:
Why is that necessary? This happens at the start of the test. If the service needs to be started for the test to work, it should use `start`. If it needs to adjust to config changes or /var/spool/ cleanup, please use `try-restart`.

Reply:
Because it listens on journal, but we "restore_file" journal and restart "systemd-journald" and it then does not see new messages in journal.

Reply:
Makes sense. Can you please add a comment?